uefi/visitor.go

```diff
@@ -0,0 +1,8 @@
+package uefi
+
+type Visitor interface {
```

Comment:
Can we put this in ast.go as well? the ASTNode interface makes little sense without the Visitor interface right there.

Reply:
I deleted `ASTNode` altogether.